docs/troubleshooting.md

```diff
@@ -99,3 +101,14 @@ If you are using Windows, the problem is likely to be either an outdated version
 of Virtual Box or you forgot to use `--vm-driver virtualbox` option when starting minishift.
 
 We recommend to use `Virtualbox >= 5.1.12` to avoid this issue.
+
+<a name="users-authentication"></a>
+## Users and authentication
+
+### Some special characters cause passwords to fail
+
+Depending on your operating system and shell environment, certain special characters
+can trigger variable interpolation and therefore cause passwords to fail.
+
+Workaround: When creating and entering passwords, wrap the string with single quotes in
+the following format: '<password>'
```

Comment:
Not working due to `<`. 

It should be  `... following format: '&lt;password>'`
![screenshot from 2017-02-20 16-42-18](https://cloud.githubusercontent.com/assets/1132451/23122811/cb2718a2-f78b-11e6-950b-f3b343b1af6a.png)